inventory/sample/group_vars/all.yml

```diff
@@ -3,6 +3,9 @@ k3s_version: v1.23.4+k3s1
 ansible_user: ansibleuser
 systemd_dir: /etc/systemd/system
 
+# Set your timezone
+system_timezone: "Your/Timezone"
```

Comment:
is it possible to default to the server time zone if one is not supplied?  I do not want to force someone to set a time zone if they do not need to.  It might lead to confusion o misconfiguration.